### ISSUE TITLE
GUAI does not run/execute even with admin access.

### ISSUE THREAD

**Issue by GUAI-fan** [liuchuanz/war3guai#1]
GUAI does not run/execute even with admin access.

**Comment by liuchuanz** [comment]
even the GuAI.exe doesn't start running?

**Comment by GUAI-fan** [comment]
yes it is still not running. it is same like before.

**Comment by liuchuanz** [comment]
Any error message? Minimize settings of the previous GuAI, and try this run this version again.

**Comment by GUAI-fan** [comment]
No, it does not give any error message when I right click on GuAI and select "Run as Administrator" it will act like the program is about to launch the hour glass will show on the mouse cursor for few seconds and then the hourglass will go away and nothing will happen, the program will not launch, and yes I tried running the new GuAI when the old GuAI are all closed. Send me your email and I will send you the video clip about what it actually happens when try to run GuAI.

**Comment by GUAI-fan** [comment]
Hi I have sent the video in proplay@163.com please check the video.

**Comment by liuchuanz** [comment]
I've viewed the video, and have no clue how this happens, I have tested it on 3 computers, it opens normally. 

Is there anything printed in the GuAI.log?
Can you try it on a different computer? 
Finally, can you try putting the GuAI.dat into the old version, and run GuAIClientLoader.exe, I wonder what would happen.

**Comment by GUAI-fan** [comment]
Okay I tried in a different computer and it works, and it seems to be like an issue with the operating system, it works in windows 7 os but doesn't run in windows 10, may be because of some security features in windows 10. Well I guess I need to downgrade my OS to windows 7.

**Comment by GUAI-fan** [comment]
Hi,
I found one problem, guai-3.8.6b.146 version works in RGC but the new one which you have recently uploaded does not work with RGC because RGC detects it every time i try to run the new GUAI is there any fix for this? 

    On Wednesday, 22 March 2017 11:12 AM, liuchuanz <notifications@github.com> wrote:
 

 I've viewed the video, and have no clue how this happens, I have tested it on 3 computers, it opens normally.Is there anything printed in the GuAI.log?
Can you try it on a different computer?
Finally, can you try putting the GuAI.dat into the old version, and run GuAIClientLoader.exe, I wonder what would happen.—

**Comment by liuchuanz** [comment]
guai-3.8.6b.146  is an verio  very long time ago, may be some later feature brought the detection. I  recommend use same/minimal settings, change the exe file name. if 3.9.1 works, you can try remove the 'alien.dll' related code, so the dll won't be loaded. 

Full anaysis and fix requires countable efforts, I can't afford it now. so the RGC detection issue may remains. you can try using other active anti-anti-cheat tools to help GuAI with it.

**Comment by GUAI-fan** [comment]
GUAI use to work in RGC but after the recent update of RGC client GUAI shows a message that says
 "[11:32:27] Unsupported game version." and no longer supports anymore is there a workaround to fix this issue? I have attached the screenshot for reference.

![untitled](https://cloud.githubusercontent.com/assets/26496593/25562080/4e2ebda2-2d99-11e7-96e7-3b7500c231a5.png)

**Comment by CalvinRui** [comment]
Bin code,release out!
